front/public/locales/fr/stdcm.json

```
@@ -42,7 +42,15 @@
       "downloadSimulationSheet": "Télécharger la fiche de simulation",
       "formatCreationDate": "{{day}}/{{month}}/{{year}} {{hours}}:{{minutes}}",
       "gesicoRequest": "et joignez ce document à votre demande GESICO DSDM",
-      "notFound": "Aucune solution n’a été trouvée pour cette configuration",
+      "notFound": "Aucune solution n'a été trouvée pour cette configuration.",
```

No point at the end in the mockup same for english trad)

```suggestion
      "notFound": "Aucune solution n'a été trouvée pour cette configuration",
```